Build/Templates/CurrentVersion.Template.cs

```diff
@@ -1,20 +1,27 @@
-namespace DynamicLanguageRuntime {{
-    internal static class CurrentVersion {{
-        public const int Major = {0};
-        public const int Minor = {1};
-        public const int Micro = {2};
-        public const string ReleaseLevel = "{3}";
-        public const int ReleaseSerial = {4};
+/*
+ * This code was generated by TemplateFile as part of the build process. Do not edit it.
+ *
+ * Generated on ${CurrentDate}.
+ *
+ */
 
-        public const string ShortReleaseLevel = "{5}";
+namespace DynamicLanguageRuntime {
+    internal static class CurrentVersion {
+        public const int Major = ${MajorVersion};
+        public const int Minor = ${MinorVersion};
+        public const int Micro = ${MicroVersion};
+        public const string ReleaseLevel = "${ReleaseLevel}";
+        public const int ReleaseSerial = ${ReleaseSerial};
 
-        public const string Series = "{0}.{1}";
-        public const string DisplayVersion = "{6}";
-        public const string DisplayName = "{9} {6}";
+        public const string ShortReleaseLevel = "${ShortReleaseLevel}";
 
-        public const string AssemblyVersion = "{0}.{1}.0.{7}";
+        public const string Series = "${MajorVersion}.${MinorVersion}";
+        public const string DisplayVersion = "${DisplayVersion}";
+        public const string DisplayName = "${ProjectName} ${DisplayVersion}";
+
+        public const string AssemblyVersion = "${MajorVersion}.${MinorVersion}.0.${AssemblyRevision}";
```

Comment:
Any idea why we ignore micro in the AssemblyVersion?

Reply:
No, I was confused by that.